apps/python/scorgls.py

```diff
@@ -70,7 +79,10 @@ def run(self):
              self.database().timeToString(self._startTime),
              self.database().convertColumnName("time_value"),
              self.database().timeToString(self._endTime))
-
+        
+        if self.author:
+            q = q + " and Origin.%s = '%s' " % (self.database().convertColumnName("creationInfo_author"), self.author)
```

Comment:
self.author should be escaped to avoid SQL injections. Sure, that runs on a database which the user has access to, but anyway it is good practice. `self.query().toString(self.author)` would to the job.

Reply:
Instructive ! Thanks.